spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe "Ingredients Index Page" do 
+  before(:each) do 
+    @ingredient1 = Ingredient.create!(name: "Ground Beef", cost: "2")
+    @ingredient2 = Ingredient.create!(name: "Salt", cost: "1")
+    @ingredient3 = Ingredient.create!(name: "Rice", cost: "4")
+  end
+  it "can display a list of all available ingredients and their cost" do 
+    visit "/ingredients"
+
+    expect(page).to have_content(@ingredient1.name)
```

Comment:
This test is good, but could be more robust with within blocks. For example, expect(page).to have_content(ingredient1.cost) is just expecting the number 2 to be somewhere on the page. What if there was a phone number on the page that contained a 2? Or a different ingredient has the same cost That could give a false positive. Within blocks are handy so that you can narrow down where you want to see the 2 (i.e. in the section of the page just for Ground Beef)